profile/README.md

```diff
@@ -0,0 +1,41 @@
+# OpenTimelineIO
+[![OpenTimelineIO](images/OpenTimelineIO@3xDark.png)](http://opentimeline.io)
```

Comment:
We could probably support both light and dark themes: https://github.blog/changelog/2021-11-24-specify-theme-context-for-images-in-markdown/ (for light theme users like me)

Reply:
Didn't know about that. I'll give it a try here and if we like the result we can backport to the main repo.